gapis/service/service.proto

```diff
@@ -671,6 +672,76 @@ message MemoryRange {
   uint64 size = 2;
 }
 
+message Metrics {
+  oneof metrics {
+    MemoryLayout memory_layout = 1;
+  }
+}
+
+// The description of the memory layout of the API state
+message MemoryLayout {
```

Comment:
Feel `MemoryLayout` name is a little bit confusing in GAPID, as we use this name for 'alignment, data width' things also. May be something like: `MemoryUsage`?

@AWoloszyn @ben-clayton @pmuetschard Any better idea or just using `MemoryLayout` is fine?

Reply:
What about 'MemoryBreakdown'?

Reply:
That is a fair point, overloading terms may not serve us well.
MemoryBreakdown is fine with me. Even a MemoryUsage (although people may expect a single number out of that).